src/interpreter/errors.ts

```diff
@@ -32,7 +34,8 @@ export class UnsupportedDeclarationError extends InterpreterError {}
 
 export enum UnsupportedOperatorErrorType {
   BINARY = 'binary ',
-  LOGICAL = 'logical '
+  LOGICAL = 'logical ',
+  UNARY = 'unary'
```

Comment:
```suggestion
  UNARY = 'unary '
```

Should there be a trailing space?